tools/check_format_test_helper.py

```diff
@@ -58,6 +61,10 @@ def getInputFile(filename):
 # code.
 def fixFileHelper(filename):
   infile = os.path.join(src, filename)
+  directory = os.path.dirname(filename)
+  if not directory == '' and not os.path.isdir(directory):
```

Comment:
doesn't `directory != ''` work? It looks like it should based on http://www.tutorialspoint.com/python/python_basic_operators.htm

Reply:
yeah that works. I'm by no means a python expert :)

Reply:
blind leading the blind here but StackOverflow is my friend :)